lms/urls.py

```diff
@@ -91,6 +91,8 @@
     # independently of courseware. https://github.com/edx/edx-val
     url(r'^api/val/v0/', include('edxval.urls')),
 
+    # Full Course/Library Import/Export API
+    url(r'^api/import_export/v1/', include('openedx.core.djangoapps.import_export.urls')),
```

Comment:
If it is controversial, then I'm happy to have it behind a feature flag, but it would be extremely helpful in automating our production processes which we share much of with edx.org, i.e. migrate a course from courses.edge to courses.edx.org, but we disable studio on our production courses site to save on resources (which edx.org could potentially do more easily with this change if they so desired).